patches/minecraft/net/minecraft/client/renderer/model/WeightedBakedModel.java.patch

```diff
@@ -15,7 +15,7 @@
  
 -   public List<BakedQuad> func_200117_a(@Nullable BlockState p_200117_1_, @Nullable Direction p_200117_2_, Random p_200117_3_) {
 -      return (WeightedRandom.func_180166_a(this.field_177565_b, Math.abs((int)p_200117_3_.nextLong()) % this.field_177567_a)).field_185281_b.func_200117_a(p_200117_1_, p_200117_2_, p_200117_3_);
-+   // FORGE: Implement our overloads (here and below) so child models can have custom logic 
++   // Forge: Implement our overloads (here and below) so child models can have custom logic 
```

Comment:
"here and below"?  i don't see a quads method below

Reply:
Yeah, this method is confusing because there's one overloaded function below, but there's two completely new methods, including getQuads itself, Added more comments to describe what is going on.